packages/autorest.go/src/generator/structs.ts

```diff
@@ -182,6 +182,9 @@ export function getXMLSerialization(prop: Property, lang: Language): string {
   if (prop.schema.serialization?.xml?.name) {
     // xml can specifiy its own name, prefer that if available
     serialization = prop.schema.serialization.xml.name;
+  } else if (prop.schema.serialization?.xml?.text) {
+    // see https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-text
+    serialization = ',chardata';
```

Comment:
Is the link supposed to explain this value? I'm still wondering about the leading comma, and what "chardata" means

Reply:
It explains where the value of `text` comes from (the `x-ms-text` extension).  The value `,chardata` is what the in-box [XML parser](https://pkg.go.dev/encoding/xml#Unmarshal) uses to indicate if a field is raw character data.  I can update the comment to clarify this.